web/vite.config.ts

```diff
@@ -10,6 +10,18 @@ export default defineConfig(() => ({
     svelte(),
     nodeResolve({
       extensions: ['.js', '.ts']
-    })
+    }),
+    {
+      name: 'html-transform',
+      enforce: 'pre' as const,
+      transformIndexHtml: (html: string) => {
+        return html.replace(
+          '/main/icon.svg',
+          process.env.NODE_ENV === 'development'
```

Comment:
It will use dev icon only in development but now in staging.

We can fix it by define `STAGING` env variable and add it to every `pnpm build` call in `.github/workflows`.

Something like:

```
          process.env.NODE_ENV === 'development' || process.env.STAGING
```

```
  run: pnpm build
  env:
    STAGING: 1
```

It will set `process.env.STANGING`.

You can try it locally by calling `STAGING=1 pnpm build` and then checking the result in `dist/index.html`

Reply:
check .yaml files in .github/workflows, is it okay?

Reply:
Looks good